### PR TITLE
Add logic for finding sold lots in most recent round

### PR DESCRIPTION
**Pull request by reginafcompton**
This PR handles issue #301.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -232,13 +236,16 @@ <h4>{{content.data.street_number}} {{content.data.street_dir}} {{content.data.st
             if (props.square_feet){
                 info += "<tr><td>Sq ft</td><td>" + LargeLots.addCommas(Math.floor(props.square_feet)) + "</td></tr>";
             }
-            
+
             // Add colorful status labels
-            if (props.status == 'sold') {
+            if ((props.status == 'sold') || (pins_sold.indexOf(props.pin_nbr) > 0)) {
```
Comment (hancush) [inline templates/index.html]: Does the second condition need to be >= 0? (Not sure if JS arrays are zero-indexed?)

File: lots/static/js/largelots_mapmaker.js [hunk 2]
```diff
@@ -170,19 +172,19 @@ var LargeLots = {
       var sql = 'select * from ' + LargeLots.cartodb_table + ' where ';
       var clauses = []
       if(checks.indexOf('sold') >= 0){
-          soldSQL = 'select * from all_sold_lots';
+          soldSQL = "(select cartodb_id, the_geom, the_geom_webmercator, pin, pin_nbr, street_name, street_direction, street_type, ward::int, community from all_sold_lots) UNION ALL (select cartodb_id, the_geom, the_geom_webmercator, pin, pin_nbr, street_name, street_direction, street_type, ward::int, community from large_lots_2016_fall_expansion where pin_nbr in (" + pins_sold + "))";
```
Comment (hancush) [inline lots/static/js/largelots_mapmaker.js]: This query looks fine! For my learning, why did it need to change?

Comment (reginafcompton) [reply lots/static/js/largelots_mapmaker.js]: The `all_sold_lots` table contains lots from the previous pilots, but not the Fall 2016 Expansion. 

To create the "purple" sold layer on the map, I had to join (UNION ALL) data from `all_sold_lots` with data from `large_lots_2016_fall_expansion` (but only pins with applicants on Step 11). 

The thing that took me some time: selecting discrete fields from Carto - the query must include `cartodb_id, the_geom, the_geom_webmercator`...i.e., you cannot execute "select pin from all_sold_lots" and expect results.

File: templates/index.html (hunk 1 above)
Comment (reginafcompton) [reply templates/index.html]: Yep, it should be >=0 (or > -1)....arrays and strings, as we have here, starting indexing at zero, of course.

**Comment by reginafcompton** [comment]
@hancush - JS evaluation ready for review!

**Pull request merged**